github.js

```diff
@@ -0,0 +1,44 @@
+import got from 'got';
+import race from 'promise-rat-race';
+
+function getChangelog(
+  {
+    githubRepo = {
+      user: '',
+      project: '',
+      path: '',
+    },
+    gitHead = 'master',
+  },
+) {
+  if (githubRepo === null) {
+    return {changelogFilename: null};
+  }
+
+  const {user, project, path} = githubRepo;
+  if (user.length < 1 || project.length < 1) {
+    return {changelogFilename: null};
+  }
+
+  const baseGithubURL = `https://raw.githubusercontent.com/${user}/${project}/${gitHead}/${`${path.replace('/tree/', '')}`}`;
+  const files = [
+    'CHANGELOG.md',
```

Comment:
NPM uses a regex for this, but we obviously can't

https://github.com/npm/npm/blob/master/lib/utils/tar.js#L81

> `changes|changelog|history`

I don't check for `changes`/`changes.md`, but I think it's not often used.